py-polars/polars/dataframe/frame.py

```diff
@@ -1792,7 +1792,11 @@ def item(self, row: int | None = None, column: int | str | None = None) -> Any:
         elif row is None or column is None:
             raise ValueError("Cannot call '.item()' with only one of 'row' or 'column'")
 
-        return self[row, column]
+        return (
+            self._df.select_at_idx(column)
+            if isinstance(column, int)
+            else self._df.column(column)
+        ).get_idx(row)
```

Comment:
Can we make the case of `item()` also call this optimized version? E.g. set `row` and `column` to 0 on line 1790.

Reply:
Maybe my LASIK is failing me; we can indeed :)
Updated accordingly!